@kiva/kv-components/utils/carousels.js

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable import/prefer-default-export */
+import {
+	computed,
+	onMounted,
+	onUnmounted,
+	ref,
+	toRefs,
+	nextTick,
+} from 'vue-demi';
+import EmblaCarousel from 'embla-carousel';
+import { throttle } from './throttle';
+
+export function carouselUtil(props, { emit, slots }, extraEmblaOptions) {
```

Comment:
This is all the current carousel code, the only change I made is added this `extraEmblaOptions` param which is used on line: 131.